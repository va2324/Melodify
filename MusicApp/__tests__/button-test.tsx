import React from 'react';
import renderer from 'react-test-renderer';
import Button from '@/components/Button';

describe('button', () => {
    test('renders successfully', () => {
        const tree = renderer.create(<Button label='button'/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});