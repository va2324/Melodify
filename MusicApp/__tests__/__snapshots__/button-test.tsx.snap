// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button renders successfully 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "height": 68,
      "justifyContent": "center",
      "marginHorizontal": 20,
      "padding": 3,
      "width": 320,
    }
  }
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "alignItems": "center",
        "borderRadius": 10,
        "flexDirection": "row",
        "height": "100%",
        "justifyContent": "center",
        "width": "100%",
      }
    }
  >
    <Text
      style={
        {
          "color": "#fff",
          "fontSize": 16,
        }
      }
    >
      button
    </Text>
  </View>
</View>
`;
